packages/veritone-react-common/src/components/IngestionJobs/NullState/styles.scss

```diff
@@ -0,0 +1,42 @@
+@import 'src/styles/modules/variables';
+
+.nullStateView {
```

Comment:
see the comment I left in the previous PR regarding the styles that are not needed here.

Reply:
fixed